pkgs/data/fonts/junicode/default.nix

```
@@ -1,20 +1,21 @@
 { lib, fetchzip }:
 
 fetchzip {
```

To enable use of `version` within the attrset, it must be made recursive (since `version` is defined withing the same attrset):

```suggestion
fetchzip rec {
```